packages/core/src/components/Modal/private/InnerContent.tsx

```diff
@@ -11,40 +12,151 @@ export type Props = {
   children: NonNullable<React.ReactNode>;
   /** Footer content. */
   footer?: React.ReactNode;
-  /** True to show the large version of the Dialog. */
+  /** Show the large version of the Dialog. */
   large?: boolean;
+  /** Show the medium version of the Dialog. */
+  medium?: boolean;
```

Comment:
Should medium be default and small be a prop?

Reply:
let me check with ken which he'd like to be the default

Reply:
okay! ken is down for medium being the default. i'll switch things up. medium as default and minimum will be through a `small` prop